### PR TITLE
Fixed function highlighting with spaces

### PR DESCRIPTION
**Pull request by Paulb23**
Fixed function highlighting with spaces mentioned in issue #5454

### CODE REVIEW

File: scene/gui/text_edit.cpp [hunk 1]
```diff
@@ -852,6 +852,11 @@ void TextEdit::_notification(int p_what) {
 								k++;
 							}
 
+							// check for space between name and bracket
```
Comment (bojidar-bg) [inline scene/gui/text_edit.cpp]: It is properly called parenthesis BTW

Comment (neikeq) [inline scene/gui/text_edit.cpp]: Yeah, brackets are either _[]_ (squared) and _{}_ (curly).

Comment (bojidar-bg) [inline scene/gui/text_edit.cpp]: @neikeq And the latter are braces, to be fair.

Comment (neikeq) [inline scene/gui/text_edit.cpp]: @bojidar-bg Both terms are valid :P http://english.stackexchange.com/a/3381

**Pull request merged**